webapp/pages/profile/_id/_slug.vue

```diff
@@ -64,20 +66,21 @@
               </client-only>
             </ds-flex-item>
           </ds-flex>
-          <ds-space margin="small">
-            <template v-if="!myProfile">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:31:47Z" title="Thursday, January 23rd 2020, 6:31:47 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -67,7 +69,7 @@
           <ds-space margin="small">
             <template v-if="!myProfile">
               <hc-follow-button
-                v-if="!user.isMuted"
+                v-if="!user.isMuted || !user.isBlocked"
```


This is equivalent to:
```js
!(user.isMuted && user.isBlocked)
```

Are you sure? This whould show the button e.g. if you muted the user but did not block the user. :confused: 

Either:
```js
!(user.isMuted || user.isBlocked)
```

Or just leave it as it was:
```js
!user.isMuted
```
in case you want to allow that you can block **and** follow a user at the same time.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:34:42Z" title="Thursday, January 23rd 2020, 6:34:42 pm +01:00">Jan 23, 2020</time>_

---


this is questionable what we want to do here cause it was the case that when a user `BLOCKED` another, that it would remove the `Follow` button and replace it with a `Unblock user` button
what should happen now? If a user has `MUTED` another user, it should appear `Unmute user`... and if they have `BLOCKED` as well?
should there be two buttons? I guess that might be noisy...

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T17:44:02Z" title="Thursday, January 23rd 2020, 6:44:02 pm +01:00">Jan 23, 2020</time>_

---


see my comment :point_down: